code/game/machinery/cryopod.dm

```diff
@@ -413,8 +413,11 @@ GLOBAL_VAR_INIT(cryopods_enabled, FALSE)
 			tgui_alert(target, "You're a Head of Staff![generic_plsnoleave_message]")
 			caught = TRUE
 		if(A)
-			tgui_alert(target, "You're a [A.name]![generic_plsnoleave_message]")
-			caught = TRUE
+			if(A.name == "valentine")
+				caught = TRUE
```

Comment:
You don't want this to be set to true here, or it'll still block from using cryo once without telling you why.